.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove the .idea directory.  Can you also add a .gitignore similar to the other projects to prevent .idea and other unwanted files in the repo?